test/methods/options.yaml

```diff
@@ -0,0 +1,2 @@
+methods: delete
```

Comment:
Shouldn't `methods` here take an `array`?

Reply:
Correct, I must have missed it.

Reply:
Fixed